python/paddle/fluid/layers/nn.py

```diff
@@ -9334,6 +9335,76 @@ def get_tensor_from_selected_rows(x, name=None):
     return out
 
 
+def shuffle_channel(x, group, name=None):
+    """
+    **Shuffle Channel Operator**
+    This operator shuffles the channels of input x.
+    It divide the input channels in each group into :attr:`group` subgroups,
+    and obtain a new order by selecting element from every subgroup one by one.
+
+    Please refer to the paper
+    https://arxiv.org/pdf/1707.01083.pdf
+    
+    .. code-block:: text
```

Comment:
```suggestion
    .. code-block:: text

```
 A blank line following .. code-block::text or .. code-block:: python  is needed to display the example contents. 


By using [preview tool](https://github.com/PaddlePaddle/Paddle/pull/15027#pullrequestreview-188400529), the result is as follows:

===============================

- **Without the blank line, the example will not be displayed** :

![blank line](https://user-images.githubusercontent.com/31058429/50535407-97178380-0b84-11e9-973e-d6cbcd998721.JPG)

===============================


- **only added with a blank line, will the example be displayed** 
![ok](https://user-images.githubusercontent.com/31058429/50535532-f9bd4f00-0b85-11e9-8d02-ef837acdd064.png)